deps/chakrashim/src/v8functiontemplate.cc

```diff
@@ -238,9 +238,9 @@ class FunctionTemplateData : public TemplateData {
       }
 
       JsValueRef function;
+      Local<String> className = !instanceTemplate.IsEmpty() ?
+	instanceTemplate->GetClassName() : Local<String>();
       {
-        Local<String> className = !instanceTemplate.IsEmpty() ?
```

Comment:
I'll revert the changes in this file; an artifact of my first attempt at adding the `toStringTag` on the proto rather than each instance.